### ISSUE TITLE
Not rendering in IE

### ISSUE THREAD

**Issue by chriswhong** [NYCPlanning/labs-bbl-list-generator#5]
The map is not rendering in IE for some reason... will post more here once I can get in front of a Windows machine.

**Comment by chriswhong** [comment]
IE11 is getting throwing syntax errors on the arrow functions... need to add `babel-standalone` and get the config right.  Working on this now.

**Comment by chriswhong** [comment]
So, the issue is that to use ES6 in-browser (without a separate build step), we have to both _transpile_ and _polyfill_.  Transpilation allows for the new syntax of ES6, polyfill allows for the new methods (like `array.find()`)

So, including both `babel-standalone` and `babel-polyfill` from CDNs is the workaround, and seems have everything working as expected in IE11.

```
    <script src="https://cdnjs.cloudflare.com/ajax/libs/babel-polyfill/6.23.0/polyfill.min.js"></script>
    <script src="https://unpkg.com/babel-standalone@6/babel.min.js"></script>
    <script src="js/layerConfig.js" type="text/babel"></script>
    <script src="js/carto.js" type="text/babel"></script>
    <script src="js/scripts.js" type="text/babel"></script>
```
Fixed in fc2398edcc9d39b8d17fee1c459b2f7171fd1db1
